caas/kubernetes/provider/specs/types.go

```diff
@@ -100,26 +98,6 @@ func (ps PodSpec) IsEmpty() bool {
 		ps.DNSPolicy == ""
 }
 
-var boolValues = set.NewStrings(
```

Comment:
we still need the bool value mapping
yes -> true
on -> true 
etc
and if the user enters 'yes', then that's treated as a string

Reply:
ah, right, no double handling naymore